modelling/src/main/java/org/axonframework/modelling/saga/AbstractSagaManager.java

```diff
@@ -191,7 +191,7 @@ public boolean supportsReset() {
     }
 
     @Override
-    public void performReset() {
+    public void performReset(Object resetInfo) {
         throw new ResetNotSupportedException("Sagas do no support resetting tokens");
```

Comment:
Hmmmm... Even if Saga `@Allow`s replaying?

Reply:
Yes, that is the stance, as that would mean you are reintroducing side effects which is an unlikely scenario.

Although it's true that several `EventHandlerInvoker`s can be paired under a `MultiEventHandlerInvoker`, thus potentially combining Sagas (which you do not want to replay) and regular Event Handling Components (EHCs), the `canHandle` method would still make certain the `AbstractSagaManager` (which is an `EventHandlingInvoker` implementation), would not be called:

```java 
@Override
public boolean canHandle(EventMessage<?> eventMessage, Segment segment) {
    return delegates.stream().anyMatch(i -> canHandle(i, eventMessage, segment));
}
private boolean canHandle(EventHandlerInvoker invoker, EventMessage<?> eventMessage, Segment segment) {
    return (invoker.supportsReset() || !ReplayToken.isReplay(eventMessage)) && invoker.canHandle(eventMessage, segment);
}
```

Hence, I still believe we're covered here.

Reply:
You are right! It's actually not possible to `@AllowReplay` on a saga which is a discussion on its own and shouldn't block this PR anyway. Thanks for the detailed explanation @smcvb!

Reply:
As discussed, I should've included the handle method in my previous description as well, as the `canHandle` does an `anyMatch`.
Thus for clarity:
```java
@Override
public void handle(EventMessage<?> message, Segment segment) throws Exception {
    for (EventHandlerInvoker i : delegates) {
        if (canHandle(i, message, segment)) {
            i.handle(message, segment);
        }
    }
}
```

You can see the `private boolean canHandle(...)` is called there, effectively assuring a Saga will not handle events upon a reset.

Reply:
Created an issue to support replay on sagas: https://github.com/AxonFramework/AxonFramework/issues/1458